intermediate_notebooks/examples/linear_regression_demo.ipynb

```diff
@@ -1,768 +1,820 @@
-            "text/plain": [
-              "<Figure size 432x288 with 1 Axes>"
-            ]
-          },
-          "metadata": {
-            "tags": []
-          }
-        }
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "8MqX73B4s5tv",
-        "colab_type": "text"
-      },
-      "source": [
-        "## Multiple Linear Regression \n",
-        "- Our mean squared error for Simple Linear Regression looks kinda high..\n",
```

Comment:
there is an extra `.` at the end of the sentence

Reply:
fixed